tests/test_services.py

```diff
@@ -221,6 +221,17 @@ def test_businesses_view_no_businesses(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, "services/businesses.html")
 
+    def test_delete_listing_view(self):
```

Comment:
This would work but does it makes sense to create a new class for the view like we do for every function?